src/sql/azdata.proposed.d.ts

```diff
@@ -134,6 +134,24 @@ declare module 'azdata' {
 		}
 	}
 
+	export interface LoadingComponentBase {
+		/**
+		* When true, the wizard will show a loading spinner at the bottom left of the footer.
+		*/
+		loading?: boolean;
+
+		/**
+		 * This sets the alert text which gets announced when the loading spinner is shown.
+		 */
+		loadingText?: string;
+
+		/**
+		 * The text to display while loading is set to false. Will also be announced through screen readers
+		 * once loading is completed.
+		 */
+		loadingCompleteText?: string;
```

Comment:
Let's change this to loadingCompletedText - that way in the future when this makes it into azdata.d.ts we can have LoadingComponent extend this for these properties instead of declaring its own.

Reply:
I guess, it still does not hurt to follow the same naming convention even if we do not extend this interface for loadingComponent.